src/services/BackendService/BackendService.spec.ts

```diff
@@ -86,12 +87,14 @@ expect.extend({
 
 function generateRandomKeys(numberOfKeys: number) {
   const keys: TemporaryExposureKey[] = [];
+  const interval = 1000 * 60 * 60 * 24;
+  const today = (Math.floor(Date.now() / interval) * interval) / 1000;
   for (let i = 0; i < numberOfKeys; i++) {
     keys.push({
-      keyData: '',
-      rollingPeriod: i,
-      rollingStartIntervalNumber: i,
-      transmissionRiskLevel: 0,
+      keyData: crypto.randomBytes(16).toString('hex'),
+      rollingPeriod: 144,
+      rollingStartIntervalNumber: Math.floor((today - i * 86400) / (60 * 10)),
```

Comment:
Could you use `24 * 60 * 60` instead of `86400`? Not a huge deal but this would make it clearer to me how the math works.